hedyorg/hedy
# Don't show achievements for the first x programs ran

**Description**
As suggested by @OnnoEbbens we throw a lot of achievements to the users when they just start using Hedy. In this PR we prevent this by setting an `achievements threshold`, currently set to `5`. When the run count is lower than the threshold no run-related achievements are shown. This should fix the issue.

**Note**
I personally would like to remove the "run 1 program" achievement as this no longer makes sense getting after running x programs. However, this introduces quite some complications to fix, might not be worth the effort.

**Fixes**
This PR fixes #3734.

**How to test**
Create a new account, verify that you don't get any run-related achievements for the first 5 programs ran.
